client/errors.go

```diff
@@ -166,6 +166,7 @@ var (
 	encAuthRegex   = regexp.MustCompile(`(\s)(Encoded authorization failure message:)\s[A-Za-z0-9_-]+`)
 	userRegex      = regexp.MustCompile(`(\s)(is not authorized to perform: .+ on resource:\s)(user)\s.+`)
 	s3Regex        = regexp.MustCompile(`(\s)(S3(Key|Bucket))=(.+?)([,;\s])`)
+	snapshotRegex  = regexp.MustCompile(`(\sThe )(.+ )'(.+?)'( does not exist)`)
```

Comment:
so it's not "snapshot" any more and this is very aggressive. and already merged. @roneli

Reply:
@disq This change is a follow up on https://github.com/cloudquery/cq-provider-aws/pull/821#discussion_r863490394

Reply:
@erezrokah that's the reason of the mention in my comment. it's still very aggressive and could gobble up whole messages as long as they fit the pattern.

Reply:
I'm happy to revert that change pending @roneli's response

Reply:
Need a smaller regex I think. Maybe instead of `(.+ )` do something like `([A-Za-z0-9- ]+) `

Reply:
Great suggestions. Done in https://github.com/cloudquery/cq-provider-aws/pull/827